connectors/utils.py

```diff
@@ -677,12 +677,25 @@ def __init__(self):
         ) as file:
```

Comment:
I must have missed the addition of this class, but I do think `ExtractionService` should be instantiated with the config, instead of reading it from a hardcoded path. But we can address it in a separate PR

Reply:
This is what I attempted to do originally, but it involved too much invasive changes to the connector framework. The best option would be for this to be a stand-alone service but that was out of scope for 8.9, so I used this temporary measure. I also clarified in projects sync that it was okay to read the config file here (only for 8.9).

Reply:
Noted.